### ISSUE TITLE
add commandline switch to select adapater

### ISSUE THREAD

**Issue by shaynem** [katlogic/bindip#1]
Is it possible to add a switch so you can choose the adapter or "IP" address the application goes out rather then metric ?

**Comment by katlogic** [comment]
Yes. If you're writing your own GUI, you need to manipulate registry - see https://github.com/katlogic/bindip#faq
